app/src/main/java/com/udacity/googleindiascholarships/ui/MainActivity.java

```diff
@@ -35,6 +34,7 @@
 
     Spinner spCourses;
     ImageView ivNavHeader;
+    private final String TAG = MainActivity.class.getSimpleName();
```

Comment:
its not getting used.

Reply:
Should i have to remove this. Or may this used in a future

Reply:
Remove it